js/wp-seo-shortcode-plugin.js

```diff
@@ -0,0 +1,281 @@
+/* global console */
```

Comment:
I think it is much safer to ignore specific `console.log` lines when they occur because that way you still keep the safeguard that `jshint` will complain about `console.log` lines.